### ISSUE TITLE
Treat missing values as NA rather than the empty string ""

### ISSUE THREAD

**Issue by mstrimas** [cboettig/birddb#6]
Currently missing values in the dataset are being imported as empty strings "". Would be useful to treat them as NAs instead. Wasn't able to figure out how to do this with arrow.

**Comment by cboettig** [comment]
good call.  Yeah, arrow's docs are unclear to me too.  Looking at https://cran.r-project.org/web/packages/arrow/arrow.pdf, the argument might be `null_strings = ""` in our `open_dataset()` calls?  But from https://arrow.apache.org/docs/r/reference/read_delim_arrow.html, it looks like the argument is called `na` (like in `readr`), and already defaults to `na=c("", "NA")`.  How are missing values coded in the original  .txt.gz dump file?  as empty characters like `""`?

**Comment by mstrimas** [comment]
The text files stores them as empty, unquote strings. I did try `null_strings = ""`, but had no luck. Using `na = ""` in `arrow::open_dataset()` gives the following interesting error:

```
Error: The following option is supported in "read_delim_arrow" functions but not yet supported here: "na"
```

suggesting that maybe the intention is to implement this in future versions.
